code/modules/medical/genetics/bioHolder.dm

```diff
@@ -263,6 +246,19 @@ var/list/datum/bioEffect/mutini_effects = list()
 		special_hair_3_color_ref = toCopy.special_hair_3_color_ref
 		special_hair_3_offset_y = toCopy.special_hair_3_offset_y
 
+	proc/CopyOtherCustomizationAppearance(var/datum/appearanceHolder/toCopy)
+		if (!src.CanCopyCustomization(toCopy))
+			return
+
+		for(var/i in 1 to length(src.customizations))
```

Comment:
safe for now since we can assume both appearanceHolders will have the same # of customizationHolders, but will need a more robust soln in the future

Reply:
Ordered list coming right up